diff --git a/stan/lib/stan_math/prim/fun/asinh.hpp b/stan/lib/stan_math/prim/fun/asinh.hpp
index cc8d0d0254..534c247f07 100644
--- a/stan/lib/stan_math/prim/fun/asinh.hpp
+++ b/stan/lib/stan_math/prim/fun/asinh.hpp
@@ -60,9 +60,10 @@ namespace internal {
  */
 template <typename V>
 inline std::complex<V> complex_asinh(const std::complex<V>& z) {
+  using std::asinh;
   std::complex<double> y_d = asinh(value_of_rec(z));
-  auto y = log(z + sqrt(1 + z * z));
-  return copysign(y, y_d);
+  auto y = stan::math::log(z + stan::math::sqrt(1 + z * z));
+  return math::copysign(y, y_d);
 }
 }  // namespace internal
 }  // namespace math
