--- a/make/program
+++ b/make/program
@@ -56,7 +56,7 @@ endif
 	$(LINK.cpp) $(subst \,/,$*.o) $(CMDSTAN_MAIN_O) $(LDLIBS) $(LIBSUNDIALS) $(MPI_TARGETS) $(TBB_TARGETS) $(subst \,/,$(OUTPUT_OPTION))
 	$(RM) $(subst  \,/,$*).o
 ifeq ($(OS),Windows_NT)
-ifneq ($(notdir $(shell where tbb.dll)),tbb.dll)
+ifeq (,$(findstring tbb.dll, $(notdir $(shell where tbb.dll))))
 	@echo 'Intel TBB is not in PATH.'
 	@echo 'Consider calling '
 	@echo '$(HELP_MAKE) install-tbb'
