diff --git a/make/program b/make/program
index d9f8b3d..dbe595c 100644
--- a/make/program
+++ b/make/program
@@ -82,6 +82,7 @@ endif
 	$(LINK.cpp) $(subst \,/,$*.o) $(CMDSTAN_MAIN_O) $(LDLIBS) $(SUNDIALS_TARGETS) $(MPI_TARGETS) $(TBB_TARGETS) $(subst \,/,$(OUTPUT_OPTION))
 
 ifeq ($(OS),Windows_NT)
+ifneq(,$(TBB_TARGETS))
 ifeq (,$(findstring tbb.dll, $(notdir $(shell where tbb.dll))))
 	@echo 'Intel TBB is not in PATH.'
 	@echo 'Consider calling '
@@ -94,6 +95,7 @@ ifeq (,$(findstring tbb.dll, $(notdir $(shell where tbb.dll))))
 	fi
 endif
 endif
+endif
 
 $(patsubst %.cpp,%$(STAN_FLAGS).d,$(CMDSTAN_MAIN)) : $(CMDSTAN_MAIN)
 	$(COMPILE.cpp) $(DEPFLAGS) $<
